lib/qiita/markdown/filters/user_input_sanitizer.rb

```diff
@@ -49,6 +55,14 @@ def filter_attribute(attr, pattern)
             end.join(DELIMITER)
           end
 
+          def filter_script
+            if node["src"] == CODE_PEN_SCRIPT_URL
+              node.children.unlink
+            else
+              node.unlink
```

Comment:
The responsibility of `AttributeFilter` is to filter attributes but `#filter_script` unlinks `<script>` nodes.  IMO, you should add a new transformer class and move this method to it according to [SRP](https://qiita.com/gomi_ningen/items/02c42e2487d035f9c3c8).